src/screens/Settings/index.tsx

```diff
@@ -194,43 +194,47 @@ function Settings() {
         'requestSettings',
         appName
       )
-      setAutoSyncSaves(config.autoSyncSaves || false)
-      setUseGameMode(config.useGameMode || false)
-      setShowFps(config.showFps || false)
-      setShowOffline(config.offlineMode || false)
-      setAudioFix(config.audioFix || false)
-      setShowMangoHud(config.showMangohud || false)
+      setAutoSyncSaves(config.autoSyncSaves)
+      setUseGameMode(config.useGameMode)
+      setShowFps(config.showFps)
+      setShowOffline(config.offlineMode)
+      setAudioFix(config.audioFix)
+      setShowMangoHud(config.showMangohud)
       setDefaultInstallPath(config.defaultInstallPath)
       setWineVersion(config.wineVersion)
       setWinePrefix(config.winePrefix)
       setWineCrossoverBottle(config.wineCrossoverBottle)
       setOtherOptions(config.otherOptions)
       setLauncherArgs(config.launcherArgs)
-      setUseNvidiaPrime(config.nvidiaPrime || false)
+      setUseNvidiaPrime(config.nvidiaPrime)
       setEgsLinkedPath(config.egsLinkedPath || '')
       setEgsPath(config.egsLinkedPath || '')
-      setExitToTray(config.exitToTray || false)
-      setStartInTray(config.startInTray || false)
-      setDarkTrayIcon(config.darkTrayIcon || false)
-      setDiscordRPC(config.discordRPC || false)
-      setAutoInstallDxvk(config.autoInstallDxvk || false)
-      setAutoInstallVkd3d(config.autoInstallVkd3d || false)
-      setEnableEsync(config.enableEsync || false)
-      setEnableFsync(config.enableFsync || false)
-      setEnableFSR(config.enableFSR || false)
+      setExitToTray(config.exitToTray)
+      setStartInTray(config.startInTray)
+      setDarkTrayIcon(config.darkTrayIcon)
+      setDiscordRPC(config.discordRPC)
+      setAutoInstallDxvk(config.autoInstallDxvk)
+      setAutoInstallVkd3d(config.autoInstallVkd3d)
+      setEnableEsync(config.enableEsync)
```

Comment:
With options like that. that are not created on the config file by default,  It will `setEnableesync` with which value?
It starts as false, but then it will set as `undefined` no?

Reply:
Need to check this, once more.

From my understanding, because you told me alot of times, undefined == false casted on a bollean.
So if the functions set... expect a boolean then undefined leeds to false.

Reply:
yes, but when you check a condition, then it is a `falsy` value. Here it is setting the value to undefined. 
If you leave it like that it will store the initial value as undefined and might break something in the settings page.